cmd/swarmd/main.go

```diff
@@ -126,6 +128,12 @@ var (
 				return err
 			}
 
+			resources := &api.Resources{ScalarResources: map[string]float64{}}
+			err = flagparser.ParseUserDefinedResources(cmd.Flags(), resources, "resources", false)
```

Comment:
Not very sure. For example, Flag [`log-level`](https://github.com/docker/swarmkit/blob/master/cmd/swarmd/main.go#L40) also parses input and throws err in this file.